File: Semester 1/Programming in C&C++/Assignment 5/a5_p1.c
/*
CH-230-A
a5_p1.c
Faraz Ahmad
fahmad@jacobs.university.de
*/
#include <stdio.h>
void print(int n, char ch);
int main()
{
    int n;
    char ch;

    //taking input from user
    scanf("%d", &n);
    scanf(" %c", &ch);
    getchar();

    //calling function to print tiangle
    print(n, ch);
    return 0;
}
/**
 * @brief this function prints a triangle of height n
 * and of char c
 * 
 * @param n 
 * @param ch 
 */
void print(int n, char ch)
{
    int N = n;
    for (int i = 0; i < n; i++)
    {
        N = n - i;
        while (N != 0)
        {
            printf("%c", ch);
            N--;
        }
        printf("\n");
    }
}

File: semester_5/programming_in_python/a5/a5_p1.py
# JTMS-14
# a5_p1.py
# Faraz Ahmad
# fahmad@jacobs-university.de

# convert volume (gallons and cups) to liters
def to_liter(gallon, cup):
    gallon_to_liter = 3.7854
    cup_to_liter = 0.2366
    total_liters = (gallon * gallon_to_liter) + (cup * cup_to_liter)
    return total_liters

# Read the number of gallons and cups from the user
gallon_input = float(input("Enter the number of gallons: "))
cup_input = float(input("Enter the number of cups: "))

total_liters = to_liter(gallon_input, cup_input)

print(f"The total volume is {total_liters} liters")
